spec/api/commentable_spec.rb

```diff
@@ -32,6 +32,11 @@ def thread_result(commentable_id, params={})
         threads.index{|c| c["body"] == "can anyone help me?"}.should_not be_nil
         threads.index{|c| c["body"] == "it is unsolvable"}.should_not be_nil
       end
+      it "returns standalone threads if explicitly requested" do
+        threads = thread_result "question_1", context: "standalone"
```

Comment:
Are you required to pass along a context of standalone for this to work? It doesn't look like we're doing that in the front end yet, but I'm guessing that's going to be part of the work for actually implementing support for inline discussions on the front end?